waffle_hub/hub/adapter/tx_model/tx_model_hub.py

```diff
@@ -204,6 +204,10 @@ def get_metrics(self) -> list[dict]:
 
         return metrics
 
+    @property
+    def default_values(self):
```

Comment:
`default_params` would be better choice 😊

Reply:
done